mcpserver/tools/posts.go

```diff
@@ -170,9 +170,28 @@ func (p *MattermostToolProvider) toolReadPost(mcpContext *MCPToolContext, argsGe
 	// Format the response
 	var result strings.Builder
 	if channelName != "" && teamName != "" {
-		result.WriteString(fmt.Sprintf("Channel: %s (Team: %s)\n\n", channelName, teamName))
+		result.WriteString(fmt.Sprintf("Channel: %s (Team: %s)\n", channelName, teamName))
 	}
 
+	// Add Channel ID and Root ID to header
+	if len(posts) > 0 {
+		result.WriteString(fmt.Sprintf("Channel ID: %s\n", posts[0].ChannelId))
+
+		// Find any post with a non-empty RootId - all replies share the same RootId
+		var rootID string
+		for _, post := range posts {
+			if post.RootId != "" {
+				rootID = post.RootId
+				break
+			}
+		}
```

Comment:
My understanding is that we will have either:
1. One post
2. A list of posts, sorted by creation date. 

So could we just:
1. Set the rootID when we loop through and sort earlier
2. Just grab the newest post's rootID